tasks/config/karma.js

```diff
@@ -49,6 +53,23 @@ module.exports = function (grunt) {
           { type: 'text-summary' },
         ]
       }
-    }
+    },
   };
+
+  times(TOTAL_CI_SHARDS, i => {
+    const n = i + 1;
+    config[`ciShard-${n}`] = {
+      singleRun: true,
+      options: {
+        files: [
+          'http://localhost:5610/bundles/commons.bundle.js',
+          `http://localhost:5610/bundles/tests.bundle.js?shards=${TOTAL_CI_SHARDS}&shard_num=${n}`,
+          'http://localhost:5610/bundles/commons.style.css',
+          'http://localhost:5610/bundles/tests.style.css'
+        ]
+      }
+    };
+  });
+
+  return config;
```

Comment:
+1 to this, though I think the explanation of how we handle shards shouldn't be leaking implementation details like murmur3 or that we're overriding window.describe.  Instead, this should just explain the behavior that a person reviewing the test run would expect to observe.